src/Sidekick.Modules.Trade/Settings/AutomaticPriceCheckEditor.razor

```diff
@@ -0,0 +1,28 @@
+@using Sidekick.Common.Blazor.Settings
+@using Sidekick.Common.Settings
+
+<FormCheckbox Value="@Enabled"
+              ValueChanged="@ValueChanged">
+    @Resources.Automatic_Price_Check
```

Comment:
Use the string instead. We decided to stop using the `.cs` files properties. They exist only to give the namespace now.